aim/sdk/adapters/pytorch_lightning.py

```diff
@@ -3,10 +3,19 @@
 from argparse import Namespace
 
 try:
-    from pytorch_lightning.loggers.base import (
-        LightningLoggerBase,
-        rank_zero_experiment,
-    )
+    import pytorch_lightning as pl
+
+    if pl.__version__ < "1.7":
```

Comment:
what about using packaging.version for comparison of versions?
```py 
from packaging import version
version.parse(pl.__version__) < version.parse("1.7.0")
```

Reply:
I was trying to avoid adding additional requirements. So I guess that decision is up to you guys. If you would prefer it done this way, I will update the pull request.

Reply:
Yea, it seems packaging is not included in PSL. I just worry string comparison will introduce potential issues, e.g. `'1.17.1' < '1.7.1'` will return `True`. What about cloning `Version` class from packaging lib to Aim source code? Or there might be better options. Thoughts?

Reply:
Good point. There are also many other possible edge cases depending on how strict they are with their versioning scheme.  The best way to ensure compatibility would be to use the packaging library. See this [question](https://stackoverflow.com/questions/11887762/how-do-i-compare-version-numbers-in-python) for more options.

Reply:
@djwessel sorry for a late reply. It seems converting to a tuple and then comparing tuples is a better way to go. Except for there might be some exceptions with comparison of alpha/release/dev version. I would go with this option probably.  thoughts?

```py
def versiontuple(v):
    return tuple(map(int, (v.split("."))))

>>> versiontuple("2.3.1") > versiontuple("10.1.1")
False
```